test/studies/lcals/RunVectorizeOnlyRawLoops.chpl

```diff
@@ -471,7 +470,8 @@ module RunVectorizeOnlyRawLoops {
             var val = 0.0;
             ltimer.start();
             for 0..#num_samples {
-              forall i in vectorizeOnly(0..(len-1):int(32)) with (+ reduce sumx) {
+              // TODO: this should have a reduce intent instead of a ref intent
+              foreach i in 0..(len-1):int(32) with (ref sumx) {
```

Comment:
It looks like this .chpl file is named after using vectorizeOnly.  Does that mean it would be better to just remove the file (and .notest) entirely?

Reply:
Or rename it?

Reply:
The file is named after `vectorizeOnly` and I would be ok renaming it, but I don't want to delete it. This part of the test doesn't actually run in nightly right now, but I have another PR (blocked by other work for now) that will re-enable it. I think its important to have a test that compares the vectorization performance of `for` and `foreach`